### DISCUSSION TITLE
Ridiculously high z index when using suspense fallback component

### DISCUSSION THREAD

**Discussion by clicktodev** [pmndrs/drei#1287] [Q&A]
- `three` version: 0.148.0
- `@react-three/fiber` version: 8.11.0
- `@react-three/drei` version: 9.56.20
- `node` version: 16
- `npm` (or `yarn`) version: 8

### Problem description:

Recently added a loader to show progress % while the canvas is loading.
This however added an extremely high z-index value which broke my ui because I have a dialog that displays on top of the canvas. the dialog now has a lower z value than the canvas.
This is the value that's being applied after adding the suspense progress loader `z-index: 8388635;`

### Relevant code:

```js
function Loader() {
	const { progress } = useProgress();

	return <Html center>{Math.floor(progress)}%</Html>;
}

export default function ThreeExperience() {
	const texture = useLoader(LUTCubeLoader, "/lut/F-6800-STD.cube");

	return (
		<Canvas
			orthographic
			gl={{ antialias: true }}
			camera={{ position: [0, 0, 100], zoom: 70 }}
		>
			<Suspense fallback={<Loader />}>
				<color attach="background" args={["black"]} />
				<Scene />
				<EffectComposer disableNormalPass>
					<Bloom
						mipmapBlur
						levels={9}
						intensity={1.5}
						luminanceThreshold={1}
						luminanceSmoothing={1}
					/>
					<LUT lut={texture} />
				</EffectComposer>
			</Suspense>
		</Canvas>
	);
}

```
A preview of the z values being applied to the canvas and the loader progress div:
![image](https://user-images.githubusercontent.com/34947993/218233133-e94ba6e8-8b6d-4fa0-ad30-9c4407b2fd1f.png)

no z index when i remove the suspense loader:
![image](https://user-images.githubusercontent.com/34947993/218233264-f88b504d-d56d-417e-886a-0e0318ee07b6.png)

### Suggested solution:

the z-index attribute is removed when I remove the suspense loader component.
Also why is the z-index so high?
fixed this for now by increasing my dialog z to be larger than the one produced by react three fiber, but I'm trying to understand why the z values being generated are so high.

**Comment by drcmda** [comment]
you can set the z-index range, look into the docs for html.

anyway, html is meant for annotations. i would much rather use tunnel-rat or simple state updates to inform a parent dom component about loading states instead of using drei/html.
